src/components/common/dialogs/showSupportDialog.js

```diff
@@ -0,0 +1,26 @@
+//@flow
+import React from 'react'
+import Text from '../view/Text'
+
+export const showSupportDialog = (showErrorDialog, hideDialog, screenProps, errorCode: string) => {
+  showErrorDialog('Something went wrong on our side...' + ` Error: ${errorCode}`, undefined, {
+    boldMessage: (
+      <>
+        <Text fontWeight={'inherit'} color={'inherit'}>
+          Please contact
+        </Text>{' '}
```

Comment:
Try to not print chars outside of a `Text` component. You can add it next to "Please contact"`{'Please contact '}`